documentation/src/object-Initializer.md

```diff
@@ -0,0 +1,14 @@
+# Initializer
+
+## Index Initializers
+
+Use index initializers with dictionary.
```

Comment:
I'd add a link to the official documentation:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers#object-initializers-with-collection-read-only-property-initialization

Reply:
a307d749dc25dd00af008083a96e8e6b9891d827

Reply:
The link is declared but never used in the text...